src/wikipedia.ts

```diff
@@ -0,0 +1,26 @@
+// Use the wikipedia API directly (libraries were disappointing)
+// We use the prop: 'extracts' which has nice sections for chunking
+export const fetchDocForTerm = async (term: string): Promise<string | null> => {
+  const result = await fetch(
+    'https://en.wikipedia.org/w/api.php?' +
+      new URLSearchParams({
+        action: 'query',
+        format: 'json',
+        titles: term,
+        prop: 'extracts',
+        explaintext: 'true',
+      }),
+    {
+      headers: {
+        'content-type': 'application/json;charset=UTF-8',
+      },
+    }
+  );
+  const jsonResult = await result.json();
+  const pages = jsonResult.query?.pages;
```

Comment:
Is this supposed to always return N pages, or 1. If N, this is just going to always take the first?